src/@itwinjs-sandbox/hooks/useSampleWidget.ts

```diff
@@ -7,25 +7,18 @@ import { RegisteredRuleset, Ruleset } from "@bentley/presentation-common";
 import { Presentation } from "@bentley/presentation-frontend";
 import * as HILITE_RULESET from "@bentley/presentation-frontend/lib/presentation-frontend/selection/HiliteRules.json";
 import { UiItemsManager } from "@bentley/ui-abstract";
-import { useActiveFrontstageDef } from "@bentley/ui-framework";
 import { SampleWidgetProvider } from "@itwinjs-sandbox/components/imodel-selector/SampleWidgetProvider";
 import { defaultIModelList } from "@itwinjs-sandbox/constants";
 import { SampleIModels } from "@itwinjs-sandbox/SampleIModels";
-import { FloatingWidgetsManager } from "@itwinjs-sandbox/widgets/FloatingWidgets";
+// import { FloatingWidgetsManager } from "@itwinjs-sandbox/widgets/FloatingWidgets";
```

Comment:
nope, will remove

Reply:
> > There were times when the sample widget would not populate. After docking it or reloading the sample, it would be fine, but still happened around 1 in 5 times I opened a sample.
> 
> This issue is known and being worked on. The widget is being created and populated, it's size is just incorrect. You can manually resize to see the info, but it shouldn't be 'minimized' in the first place.

Stefan is amazing and fixed this the moment I made the comment.